nixos/modules/services/monitoring/prometheus/default.nix

```diff
@@ -76,8 +69,8 @@ let
     "--storage.tsdb.path=${workingDir}/data/"
     "--config.file=${
       if cfg.enableReload
-      then prometheusYmlOut
-      else "/run/prometheus/prometheus-substituted.yaml"
+      then "/etc/prometheus/prometheus.yaml"
```

Comment:
Also how does the new secrets work? Can we add documentation for this?

Reply:
I added the alternatives to the release notes and to the `mkRemovedOptionModule` error message.